src/views/Group.js

```diff
@@ -250,10 +286,10 @@ function Group() {
                                     <TableCell></TableCell>
                                     <TableCell align="right"><ToggleButtonGroup size="small"><ToggleButton value="add" onClick={addGroup}><SaveIcon /></ToggleButton><ToggleButton value="close" onClick={handleAddRowClose}><CloseIcon /></ToggleButton></ToggleButtonGroup></TableCell>
                                 </TableRow>
-                                {groupContext._embedded.map((group) =>
+                                {groupContext._embedded.map((group) => 
                                     <TableRow key={group_id}>
-                                        <TableCell style={{ fontSize: '1.1rem', width: '30%' }} component="th" scope="row">{group[0]}</TableCell>
-                                        <TableCell style={{ fontSize: '1.1rem', width: '30%', textAlign: 'center' }} component="th" scope="row">{group[1]}</TableCell>
+                                        <TableCell style={{ fontSize: '1.1rem', width: '30%' }} component="th" scope="row">{isLoading ? <Skeleton width="150px" /> : group[0]}</TableCell>
+                                        <TableCell style={{ fontSize: '1.1rem', width: '30%' }} component="th" scope="row">{isLoading ? <Skeleton variant="text" width="150px" /> : group[1]}</TableCell>
```

Comment:
not sure...  do you want fixed widths here?    or something percentage based?

does that behave as desired on a tablet?   small window?

Reply:
hmm good question, I haven't thought of the tablet/mobile compatibility. I think I set that to 150px to match the max tablecell width. But I agree percentage will work better on tablets(etc..), I can make the changes. Also this reminds me of other components that needs to be tuned to be mobile-friendly.

Reply:
Changed fixed width(150px) to percentage of a tablecell width.